pkgs/top-level/python-packages.nix

```diff
@@ -6038,6 +6038,29 @@ let
     };
   };
 
+  django_redis = makeOverridable ({ django ? self.django }: buildPythonPackage rec {
```

Comment:
Well other than going on a mass-renaming and breaking loads of peoples stuff, I don't know what to do. Last time I did a reasonably sized mass-change of django stuff it got pretty flatly ignored (https://github.com/NixOS/nixpkgs/pull/8986) I'm guessing because nobody wanted to grapple with the potential ramifications it may have, so I would fully expect the same to happen again.